src/services/RoleService.js

```diff
@@ -0,0 +1,28 @@
+import { api, getDefaultHeaders, manageError } from 'boot/axios';
+
+/**
+ * Get all roles.
+ * @returns {Promise<object[]>} Promise with an array of roles on success
+ * otherwise an error.
+ */
+export async function find() {
```

Comment:
find what ? please be more specific (event if we know we are inside RoleService so it's related to roles)

Reply:
see with you, updated.